src/WhereConditions/WhereConditionsBaseDirective.php

```diff
@@ -67,6 +85,43 @@ function ($builder) use ($orConnectedConditions): void {
         return $builder;
     }
 
+    /**
+     * @param QueryBuilder|EloquentBuilder $builder
+     * @param array<string, mixed>|null $condition
+     */
+    public function handleHasCondition($builder, Model $model, string $relation, ?array $condition = null, ?int $amount = null, ?string $operator = null): void
+    {
+        $additionalArguments = [];
+
+        if ($operator) {
```

Comment:
What if `$operator` is `''`?

Reply:
Then it is the same as false, and just uses the default operation of Laravel.

Reply:
Let's change this to `!== null` as well. Passing `''` would then likely cause an error, which is preferrable to silently discarding it.